app/assets/v2/js/pages/bounty_details2.js

```diff
@@ -254,33 +299,6 @@ Vue.mixin({
         }
       }
     },
-    contactFunder: function() {
-      let vm = this;
-      let text = window.prompt('What would you like to say to the funder?', '');
-
-      if (text === null) {
-        return;
-      }
-      document.location.href = `${vm.bounty.url}?admin_contact_funder=${text}`;
-    },
-    snoozeeGitbot: function() {
-      let vm = this;
-      let text = window.prompt('How many days do you want to snooze?', '');
-
-      if (text === null) {
-        return;
-      }
-      document.location.href = `${vm.bounty.url}?snooze=${text}`;
-    },
-    overrideStatus: function() {
-      let vm = this;
-      let text = window.prompt('What new status (valid choices: "open", "started", "submitted", "done", "expired", "cancelled", "" to remove override )?', '');
-
-      if (text === null) {
-        return;
-      }
-      document.location.href = `${vm.bounty.url}?admin_override_satatus=${text}`;
-    },
```

Comment:
why are we removing these actions? that will be undefined in the html vue click

Reply:
@octavioamu so those buttons don't work for ETH bounties as well!
Hence I made sure only the buttons which worked were being ported over

Reply:
These actions don't work in live ! (haven't in a long time)
https://gitcoin.co/issue/gitcoinco/web/6486/4252

So removed them out from the JS and the html for cross-chain

Reply:
I'll re-add the contactFunder and snooze bot but the override status -> let's not do that ?

Reply:
but eth bounties are not using this code

Reply:
the override status is something they use to manage hackathon bounties sometimes I believe

Reply:
lol that doesn't work :P cause that updates `idx_status` and not `bounty_state`

we'll fix this up if they ever bring it up 📦